tests/ui/resolve/issue-82156.stderr

```diff
@@ -1,8 +1,13 @@
-error[E0433]: failed to resolve: there are too many leading `super` keywords
+error[E0433]: cannot find item `super` in this scope
   --> $DIR/issue-82156.rs:2:5
    |
 LL |     super();
-   |     ^^^^^ there are too many leading `super` keywords
+   |     ^^^^^ can't use `super` as an identifier
```

Comment:
The previous error was sort of more correct, it's not a syntactic error, the path just cannot be resolved correctly, because we are already in the root module so there's no `super`.
(And it it was resolved, it would be an "expected function, found module" error.)